examples/basics/plotting/grid_holed.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
++---+-------+---+
+| 1 |   1   | 2 |
+|---+-------+---+
+| 3 | Empty | 2 |
++---+-------+---+
+| 3 |   4   | 4 |
++---+-------+---+
+"""
+
+import sys
+
+from vispy import scene, app
+
+canvas = scene.SceneCanvas(keys='interactive')
+canvas.size = 600, 600
+canvas.bgcolor = "#000000"
+canvas.show()
+
+grid = canvas.central_widget.add_grid()
+
+
+widget_top_left = grid.add_widget(row=0, col=0, col_span=2)
```

Comment:
can you add (or use if it exists already) the `bgcolor` kwarg? It would make this example even simpler / shorter